docs/content/reference/cli/rad_credential_register_azure.md

```diff
@@ -33,37 +21,33 @@ Cloud providers are configured per-Radius-installation. Configuration commands w
 or the workspace specified by '--workspace' to configure Radius. Modifications to cloud provider configuration
 or credentials will affect all Radius Environments and applications of the affected installation.
 
-```
-rad credential register azure [flags]
-```
-
 ### Examples
 
 ```
 
 # Register (Add or update) cloud provider credential for Azure with service principal authentication
-rad credential register azure --client-id <client id/app id> --client-secret <client secret/password> --tenant-id <tenant id>
+rad credential register azure sp --client-id <client id> --client-secret <client secret> --tenant-id <tenant id>
+# Register (Add or update) cloud provider credential for Azure with workload identity authentication
+rad credential register azure wi --client-id <client id> --tenant-id <tenant id>
 
 ```
 
 ### Options
 
 ```
-      --client-id string       The client id or app id of an Azure service principal.
-      --client-secret string   The client secret or password of an Azure service principal.
-  -h, --help                   help for azure
-  -o, --output string          output format (supported formats are json, table) (default "table")
-      --tenant-id string       The tenant id of an Azure service principal.
-  -w, --workspace string       The workspace name
+  -h, --help   help for azure
```

Comment:
@willdavsmith - Any reason behind this change? We still have these options that are common to sp and wi right?

Reply:
I see seperate files for sp and wi below. Do we still need this markdown ?

Reply:
we still needed the command `rad credential register azure` because it now has subcommands. I think we're probably fine to merge this

Reply:
this is auto generated stuff

Reply:
Ok.